app/models/customer.rb

```diff
@@ -0,0 +1,6 @@
+class Customer < ApplicationRecord
+  has_paper_trail only: [:customer, :address, :description]
```

Comment:
Ein Feld "customer" im Modell "Customer" ist verwirrend. Besser in "name" umbenennen!